packages/fiori/src/TimelineItem.ts

```diff
@@ -91,6 +103,24 @@ class TimelineItem extends UI5Element implements ITimelineItem {
 	@property()
 	subtitleText?: string;
 
+	/**
+	 * Defines the status of the icon displayed in the `ui5-timeline-item`.
+	 * @default "None"
+	 * @public
+	 * @since 2.5.0
+	 */
+	@property()
+	status: `${TimelineItemStatus}` = "None";
+
+	/**
+	 * Defines the accessible description of the component.
+	 * @default undefined
+	 * @public
+	 * @since 2.5.0
+	 */
+	@property()
```

Comment:
Was adding this new property part of the scope?

Reply:
As discussed offline there was misunderstanding and we're reversing the introduction of the property.